app/controllers/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class UsersController < ApplicationController
+
+  def index
+    @users = User.order(created_at: :desc).page(params[:page]).per(User.default_per_page)
```

Comment:
これでもおなじ動きにならない？
`per(10)` を省略するのが目的なので、効かないのであれば何か間違っているかも？
```suggestion
    @users = User.order(created_at: :desc).page(params[:page])
```

Reply:
なりました！
paramsでUserモデル内で定義したper数を取得しているということですかね